lib/node_modules/@stdlib/string/base/replace-before-last/README.md

```diff
@@ -0,0 +1,122 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2023 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# replaceBeforeLast
+
+> Replace the substring before the last occurrence of a specified search string.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var replaceBeforeLast = require( '@stdlib/string/base/replace-before-last' );
+```
+
+#### replaceBeforeLast( str, search, replacement )
```

Comment:
@AuenKr Similarly, would you mind adding support for `fromIndex`?

This was actually intended for `substringBeforeLast` (https://github.com/stdlib-js/stdlib/commit/0aba813bb4ecb3cda3531bd4470afa2dc05c3a3a#r56763056), but was never implemented. It would be great to go ahead and implement `fromIndex` support for the API addition in this PR.

As this is a "base" API, the `fromIndex` argument should be mandatory and not an optional argument.